src/main/java/com/joss/conductor/mobile/LocomotiveConfig.java

```diff
@@ -284,6 +284,6 @@ public String getAppFullPath() {
             default:
                 throw new IllegalArgumentException("Unknown platform: " + platformName());
         }
-        return System.getProperty("user.dir") + app;
+        return (StringUtils.isEmpty(hub())) ? System.getProperty("user.dir") + app : app;
```

Comment:
At some point in the future I'd like to check if app is a local / relative request and add the current working directory that way instead of relying on hub being null (as it can be set for local tests). For now this is fine though.